packages/Popover/src/components/Content/Content.js

```diff
@@ -94,8 +108,9 @@ const Content = React.forwardRef((props, ref) => {
       style={contentStyles}
       tabIndex={isOpen ? 0 : -1}
       zIndex={content.zIndex}
+      id={isEager && !hasCardComponent ? content.ariaId : null}
```

Comment:
Why not just always apply the `id` to the `Content`, whether there's a `Card` or not? It would be nice if `Content` didn't have to care about its contents.

Reply:
Because when there's `Card` component, we need to apply the `id` on `Card` component, otherwise we apply the same id on `Content` component. `aria-describedby` should be on the node has text as child.

Reply:
It's supposed to ONLY contain text? 😳 

I tried hacking an example page and the screen reader still picks it up (Voiceover with Chrome, at least):

![Screen Shot 2019-09-20 at 3 50 05 PM](https://user-images.githubusercontent.com/14944896/65362999-8f8a8580-dbbe-11e9-9826-20e43026fcc0.png)

Did you find a guideline or recommendation or something about the id being on the leaf node?

Reply:
Let me do some research, I tested with NVDA it only works with nodes ONLY contain text

Reply:
I guess it turned out to be okay with JAWS / NVDA?

Reply:
Found a similar example(https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html) and tested with browsers. I think I messed up my voiceover on Chrome, but it works fine on Safari. This node structure also works on NVDA, so I remove the code for cloning the child and pass it down. The code looks much cleaner now.